### DISCUSSION TITLE
플레이어별 UI를 가져와서 별도의 UI를 추가하는 방법이 있을까요?

### DISCUSSION THREAD

**Discussion by ted002** [naverz/zepeto-studio-kor#419] [5. Q&A]
현재 클라이언트 자체에 캔버스를 추가하고 UI를 추가하면 이동UI에 가려지거나 점프 UI에 가려집니다.

 이는 만들어둔 별도의 UI가 생성되고 난 뒤, 플레이어의 UI가 생성되서 발생하는 문제인듯 합니다. 그래서 각 플레이어마다 별도의 UI를 코드로서 추가해주려고 하는데, 각 플레이어가 가지고있는 컨트롤러의 캔버스 내에 이 UI들을 추가해주고 싶습니다.

![질문용](https://user-images.githubusercontent.com/73985764/155286061-43c3057d-342c-4ec3-901b-2e8003c754db.png)

이미지에서 빨간 줄이 쳐저있는 부분만 멀티플레이 용 혹은 로컬플레이 용으로 변경하고싶은데, 방법이 있을까요?

**Comment by x4cf** [comment]
제페토 플레이스 자식으로 찾지말고

GameObject.Find("UIZepetoPlayerControl).gameObject;

로 찾으면 오브젝트는 가져올 수 있을 거에요 :D

**Reply by x4cf** [reply]
ZepetoPalyers에 부착하는 컨트롤러를 복사해서 수정할 수 도 있습니다 :D
<img width="1467" alt="스크린샷 2022-02-23 오후 6 04 57" src="https://user-images.githubusercontent.com/90679926/155288990-c180ef79-63ab-4be7-af9d-c8637a76f06b.png">
<img width="1467" alt="스크린샷 2022-02-23 오후 6 05 59" src="https://user-images.githubusercontent.com/90679926/155288997-acc7a93b-f484-44ff-9a28-5c894ac019fa.png">
<img width="1467" alt="스크린샷 2022-02-23 오후 6 05 56" src="https://user-images.githubusercontent.com/90679926/155289016-aaef1037-3859-42ed-912e-5dd54df34849.png">

**Reply by ted002** [reply]
앗 감사합니다!
